Cargo.lock

```diff
@@ -1,220 +1,233 @@
 # This file is automatically @generated by Cargo.
```

Comment:
The project wasn't building for me because of compile errors in edit_toml. Ran a `cargo update` and that fixed the issue. Hence this big diff.